src/main/plugin/dcat2/update-fixed-info.xsl

```diff
@@ -378,6 +321,18 @@
     </xsl:copy>
   </xsl:template>
 
+  <!-- Fill concepts with resourceType -->
+  <xsl:template match="skos:Concept" priority="10">
```

Comment:
This should probably go in thesaurus-transformation but let's do it like that for now.